github_connector/readme/ROADMAP.rst

```diff
@@ -26,3 +26,5 @@
 .. code-block:: bash
 
    ``sudo pip install PyGitHub``
+   ``sudo pip install pygount``
```

Comment:
this is a roadmap section. that is not the place to mention how to install the module.

(and this is not necessary, as it is written in the manifest.)

Reply:
Well, not sure how this comes here initially (the PyGithHub thing). It must be in INSTALL.rst. It's needed as user documentation.

Reply:
Add these lines in the file because there is no INSTALL.rst but I am going to create it with this content because it should exist.